conf/config.py

```diff
@@ -207,6 +207,9 @@ class BaseConfiguration(object):
     # repositories that should be searched for unpublished images, specifically because of EUS base images
     UNPUBLISHED_EXCEPTIONS: list[dict[str, str]] = []
 
+    # option for enabling the automatic trigger of previous versions of operators
+    PREVIOUS_REBUILDS_AUTOTRIGGER = True
```

Comment:
How about rename this to `BUNDLE_INCLUDE_PREVIOUS_REBUILDS`?

Reply:
That's a much better name! I'll update it throughout the files.